src/modules/collection/sagas.ts

```diff
@@ -134,7 +135,6 @@ export function* collectionSaga(builder: BuilderAPI, catalyst: CatalystClient) {
     try {
       const collections: Collection[] = yield call(() => builder.fetchCollections(address))
       yield put(fetchCollectionsSuccess(collections))
-      yield put(closeModal('CreateCollectionModal'))
```

Comment:
Was this modal mistakenly closed before? Because the `handleSaveCollectionSuccess` only handles the actions of saving collections.

Reply:
I think so, I checked the commit that added it and didn't seem to make sense:
[`fix: Fix BuilderAPI cyclic dependency`](https://github.com/decentraland/builder/commit/9073f52eff63452731431bd7724a854a30c4fb28)